__test__/index.test.js

```diff
@@ -1,47 +1,95 @@
-import nock from 'nock'
+/* eslint import/first: "off" */
```

Comment:
jest.mock is hoisted--but it's a little cleaner to me by placing it at the top.

Happy to tweak this.

Reply:
That's fine, thanks ☺️